defaultPnorLayoutSingleSide.xml

```diff
@@ -174,6 +174,8 @@ Layout Description
         <physicalOffset>0x961000</physicalOffset>
         <physicalRegionSize>0x100000</physicalRegionSize>
         <side>A</side>
+        <xz/>
+        <xzSize>0x100000</xzSize>
```

Comment:
Why do we have a size here at all? Currently, if you throw a file at building a pnor that's too big, everything errors out relatively gracefullly.

Why not use the same logic? If suddenly everything is too big, then it fails out.